README.md

```diff
@@ -1 +1,346 @@
+The content of the OWNERS file is going to be something like this:
+
+```
+chart:
+  name: awesome
+  shortDescription: A Helm chart for Awesomeness
+publicPgpKey: null
+users:
+  - githubUsername: <username-one>
+  - githubUsername: <username-two>
+vendor:
+  label: acme
+  name: ACME Inc.
+```
+
+The name of the chart that you are submitting must match the value in the OWNERS
+file.  The following section explains how to submit a chart source.  And the
+section followed describes how to submit a chart verification report—finally
+submitting a chart and the report together.
+
+### Chart Source
+
+The first step is to create a directory with the version number as the name. For
+example, if you are publishing the &quot;0.1.0&quot; version of the
+&quot;awesome&quot; chart, create a directory like this:
+
+```
+charts/partners/acme/awesome/0.1.0/
+```
+
+The chart could be a tarball created using the `helm package` command or a
+directory with the chart source. If it's a tarball, it can be placed directly
+under the `0.1.0` directory.
+
+```
+charts/partners/acme/awesome/0.1.0/awesome-0.1.0.tgz
+```
+
+If the chart is a directory with the chart source, create an `src` directory to
+place the chart source.
+
+Here is an example path:
+
+```
+charts/partners/acme/awesome/0.1.0/src/
+```
+
+The file structure looks like this:
+
+```
+.
+└── src
+    ├── Chart.yaml
+    ├── README.md
+    ├── templates
+    │   ├── deployment.yaml
+    │   ├── _helpers.tpl
+    │   ├── hpa.yaml
+    │   ├── ingress.yaml
+    │   ├── NOTES.txt
+    │   ├── serviceaccount.yaml
+    │   ├── service.yaml
+    │   └── tests
+    │       └── test-connection.yaml
+    ├── values.schema.json
+    └── values.yaml
+```
+
+### Chart Verification Report
+
+When submitting a chart verification report without the chart, create a
+directory with the version number as the name. For example, if you are
+publishing the `0.1.0` version of the `awesome` chart, create a directory like
+this:
+
+```
+charts/partners/acme/awesome/0.1.0/
+```
+
+Generate the report and save it under `0.1.0` with a given file name as
+`report.yaml`.
+
+### Chart with Verification Report
+
+You can also submit a chart and the report together. As mentioned in the "Chart
+Source" section, place the source or tarball under the version numbered
+directory. Similarly, as mentioned in the "Chart Verification Report" section,
+place `report.yaml` also under the save under the version numbered
+directory.
+
+## Troubleshooting Pull Request Failures
+
+After submitting the pull request, you can keep the pull request page open to
+observe the results.  If there is no issue with the pull request, it's going go
+be merged automatically.  If there is any issue with with PR, a comment with
+details about the issue will appear.
+
+### Error when submitting files not part of any chart
+
+If you try to submit changes not related to any particular chart, you will
```

Comment:
If you try to submit changes not related: If you try to submit changes that are not related